scripts/superdesk-vocabularies/vocabularies.js

```diff
@@ -73,7 +73,7 @@
         }
 
         function onSuccess(result) {
-            notify.success(gettext('Vocabulary saved succesfully'));
+            notify.success(gettext('Vocabulary saved successfully'));
```

Comment:
Or we could use 'was' since the notification takes place immediately after the save.